src/App/Fossa/Container/Test.hs

```diff
@@ -55,7 +56,9 @@ test ::
   ) =>
   ContainerTestConfig ->
   m ()
-test ContainerTestConfig{..} =
+test ContainerTestConfig{..} = do
+  _ <- ignoreDebug $ runFossaApiClient apiOpts preflightChecks
```

Comment:
looks like you are using ignoreDebug in all invocation of `runFossaApiClient apiOpts preflightChecks`, maybe something like,

```Haskell
guardWithPreflightChecks :: ApiOpts -> ....
guardWithPreflightChecks apiOpts = ignoreDebug $ runFossaApiClient apiOpts preflightChecks

preflightChecks ::
  ( Has Diagnostics sig m
  , Has (Lift IO) sig m
  , Has FossaApiClient sig m
  ) =>
  m ()
preflightChecks = context "preflight-checks" $ do
```